xml/System.Reflection.Metadata/SignatureTypeCode.xml

```diff
@@ -85,7 +85,7 @@
         <ReturnType>System.Reflection.Metadata.SignatureTypeCode</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Represents <see cref="System.Byte"/> in signatures.</summary>
```

Comment:
There should be an article for this and each of the following member descriptions -- e.g., "Repesents a Byte...."